valinta-tulos-service/src/main/resources/oph-configuration/dev-vars.yml

```
@@ -1,8 +1,10 @@
 ---
+cas_url: "https://virkailija.testiopintopolku.fi/cas/p3"
```

Tässä ei tarvita /p3 mikäli ollaan palomuurin sisällä

```suggestion
cas_url: "https://virkailija.testiopintopolku.fi/cas"
```